js/.htaccess

```
@@ -0,0 +1,16 @@
+# Apache 2.2
+<IfModule !mod_authz_core.c>
+    Order deny,allow
+    Deny from all
+    <Files ~ "(?i)^.*\.(css|js|gif|png|jpg|svg|html|map|eot|ttf|woff)$">
+        Allow from all
+    </Files>
+</IfModule>
+
+# Apache 2.4
+<IfModule mod_authz_core.c>
+    Require all denied
+    <Files ~ "(?i)^.*\.(css|js|gif|png|jpg|svg|html|map|eot|ttf|woff)$">
```

```suggestion
    <Files ~ "(?i)^.*\.(css|js|json|gif|png|jpg|svg|html|map|eot|ttf|woff)$">
```
There is the bower.json too.

---

What is it user for? I don't see any connection/usage of bower.json via Browser.